api/v1alpha7/types.go

```diff
@@ -108,10 +108,11 @@ type PortOpts struct {
 	// The virtual network interface card (vNIC) type that is bound to the neutron port.
 	VNICType string `json:"vnicType,omitempty"`
 
-	// A dictionary that enables the application running on the specified
-	// host to pass and receive virtual network interface (VIF) port-specific
-	// information to the plug-in.
-	Profile map[string]string `json:"profile,omitempty"`
+	// Profile is a set of key-value pairs that are used for binding details.
```

Comment:
The docstring should use the json name, which I'm expecting should be `profile` unless we decide to follow the change below and use `bindingProfile`.

We haven't been consistent with this to date, but I'm trying to be consistent with new docstrings. We'll need a cleanup at some point.